### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by aaronmgdr**
add Nathan as code owner for openpgpkeys 

And thought @jeanregisser might want to be a code owner for mobile. 

Also @cmcewen Is listed 7 times. wondering if that still makes sense?

### CODE REVIEW

**Review by nategraf — APPROVED** [review]

**Review by jeanregisser — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,19 +6,20 @@
...
-/packages/devutils/ @mcortesi
+/packages/dev-utils/ @mcortesi
 /packages/docs/ @timmoreton
 /packages/faucet/ @jmrossy
 /packages/helm-charts/ @nambrot @timmoreton
 /packages/metadata-crawler/ @aaitor
-/packages/mobile/ @cmcewen @jmrossy
+/packages/mobile/ @jeanregisser @cmcewen @jmrossy
 /packages/moonpay-auth/ @annakaz
 /packages/notification-service/ @annakaz @cmcewen
```
Comment (i1skn) [inline .github/CODEOWNERS]: Suggestion to add myself
```suggestion
/packages/mobile/ @jeanregisser @cmcewen @jmrossy @i1skn 
/packages/moonpay-auth/ @annakaz @i1skn 
/packages/notification-service/ @annakaz @cmcewen @i1skn 
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,28 +6,34 @@
...
 /packages/reserve-site/ @aaronmgdr
 /packages/terraform-modules-public/ @tkporter
 /packages/terraform-modules/ @tkporter
+/packages/transaction-metrics-exporter/ 
 /packages/typescript/ @cmcewen
 /packages/utils/ @jmrossy
-/packages/web/ @cmcewen @aaronmgdr
+/packages/verification-pool-api/
+/packages/verifier/
+/packages/walletkit/
```
Comment (jmrossy) [inline .github/CODEOWNERS]: These don't exist anymore, why add them back in here?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -6,28 +6,34 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/
```
Comment (jmrossy) [inline .github/CODEOWNERS]: I guess I own this. It was something hastily threw together by Trevor and I to work around TLS issues during verification on Mainnet

**Review by jmrossy — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (aaronmgdr) [reply .github/CODEOWNERS]: ah oops they directories still existed on my machine

Comment (jmrossy) [reply .github/CODEOWNERS]: ditto for transaction-metrics-exporter

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -6,21 +6,22 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ @jmrossy 
 /packages/attestation-service/ @nambrot
 /packages/azure/ @codyborn
 /packages/blockchain-api/ @jmrossy @cmcewen
```
Comment (jeanregisser) [inline .github/CODEOWNERS]:
```suggestion
/packages/blockchain-api/ @jmrossy @cmcewen @jeanregisser
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -6,21 +6,22 @@
...
 /packages/docs/ @timmoreton
 /packages/faucet/ @jmrossy
 /packages/helm-charts/ @nambrot @timmoreton
 /packages/metadata-crawler/ @aaitor
-/packages/mobile/ @cmcewen @jmrossy
-/packages/moonpay-auth/ @annakaz
-/packages/notification-service/ @annakaz @cmcewen
+/packages/mobile/ @jeanregisser @cmcewen @jmrossy @i1skn 
+/packages/moonpay-auth/ @annakaz @i1skn 
+/packages/notification-service/ @annakaz @cmcewen @i1skn
```
Comment (jeanregisser) [inline .github/CODEOWNERS]:
```suggestion
/packages/notification-service/ @annakaz @cmcewen @i1skn @jeanregisser
```

**Review by jeanregisser — COMMENTED** [review]
Added myself to 2 more projects.

File: .github/CODEOWNERS (hunk 4 above)
Comment (i1skn) [inline .github/CODEOWNERS]:
```suggestion
/packages/blockchain-api/ @jmrossy @cmcewen @i1skn 
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -6,21 +6,22 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ @jmrossy 
 /packages/attestation-service/ @nambrot
 /packages/azure/ @codyborn
-/packages/blockchain-api/ @jmrossy @cmcewen
+/packages/blockchain-api/ @jmrossy @cmcewen @i1skn
```
Comment (jmrossy) [inline .github/CODEOWNERS]:
```suggestion
/packages/blockchain-api/ @jmrossy @cmcewen @i1skn @jeanregisser
```

**Review by jmrossy — APPROVED** [review]
Ship it!

**Pull request merged**